pkg/vm/engine/tae/logtail/tools.go

```diff
@@ -193,6 +193,14 @@ func DataChangeToLogtailBatch(src *containers.BatchWithVersion) *containers.Batc
 
 // consume containers.Batch to construct api batch
 func containersBatchToProtoBatch(bat *containers.Batch) (*api.Batch, error) {
-	mobat := containers.CopyToCNBatch(bat)
-	return batch.BatchToProtoBatch(mobat)
+	rbat := new(api.Batch)
```

Comment:
We are planning to remove CopyToCNBatch here and it is also the only place that uses CopyToCNBatch. So that CopyToCNBatch can be removed totally. In this PR, CopyToCNBatch without MPool is required. Otherwise, there will be a mpool memory leak.